packages/app/src/domain/vaccine/vaccine-delivery-and-administrations-area-chart.tsx

```diff
@@ -19,7 +36,13 @@ export function VaccineDeliveryAndAdministrationsAreaChart({
 }) {
   const { siteText, formatNumber } = useIntl();
   const firstValue = first(data.values);
-  const vaccineNames = useVaccineNames(firstValue).reverse();
+  const vaccineNames = useMemo(
+    () =>
+      vaccines
+        .filter((x) => firstValue && firstValue[x] !== undefined)
+        .reverse(),
+    [vaccines, firstValue]
```

Comment:
Unnecessary to add vaccines to the dep array since it's defined outside of the function component and won't change
```suggestion
    [firstValue]
```

Reply:
Fixed